pkg/testutil/test/test.go

```diff
@@ -0,0 +1,118 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package test
+
+import (
+	"os/exec"
+	"testing"
+)
+
+// Require skips tests when executable `name` is not present in PATH.
+func Require(t *testing.T, name string) {
+	if _, err := exec.LookPath(name); err != nil {
+		t.Skipf("required executable doesn't exist in PATH: %s", name)
+	}
+}
+
+// A Fixturer is the function signature meant to be attached to a Setup or Cleanup routine for a test.Case
+type Fixturer func(data Data, helpers Helpers)
```

Comment:
`fixture` does not seem to be a verb

Reply:
Yeah, it is barbaric.
Any other name coming to your mind?
Otherwise will come up with something else.

Reply:
Changed:
- Fixturer -> Butler
- Expecter -> Manager